src/TortoiseProc/GitLogListBase.cpp

```diff
@@ -95,8 +95,16 @@ CGitLogListBase::CGitLogListBase():CHintListCtrl()
 	m_bFilterWithRegex = !!CRegDWORD(_T("Software\\TortoiseGit\\UseRegexFilter"), FALSE);
 	m_bFilterCaseSensitively = !!CRegDWORD(_T("Software\\TortoiseGit\\FilterCaseSensitively"), FALSE);
 
-	m_From = -1;
-	m_To = -1;
+	m_Filter.m_NumberOfLogsScale = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogsScale"), -1);
+	m_Filter.m_NumberOfLogs = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogs"), 0);

```

Comment:
It shares between commits/years/months/weeks.
the value coulde be 1000/1/6/13
It will be more friendly for user. (but more code.)
I you want one "m_nNumber", I will do it. :)